File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

public class UberBlack extends Car {
    
    Map<String, Map<String, Integer>> typerCarAcepted;
    ArrayList<String> seatsMaterial;
    
    public UberBlack(String license, Account driver,
    Map<String, Map<String, Integer>> typerCarAcepted, ArrayList<String> seatsMaterial){
        super(license, driver);
        this.typerCarAcepted = typerCarAcepted;
        this.seatsMaterial = seatsMaterial;

    }

}

File: Python/UberBlack.py
from car import Car

class UberBlack(Car):

    type_car_acepted = []
    seat_material = []

    def __init__(self, license, driver, type_car_acepted, seat_material):
        super().__init__(license, driver)
        self.type_car_acepted = type_car_acepted
        self.seat_material = seat_material
    